### PR TITLE
ADR on naming convention

### PR DESCRIPTION
**Pull request by arm61**

### CODE REVIEW

**Comment by arm61** [comment]
@rozyczko the only ADR to be reviewed is `adr0001.rst`

File: docs/adrs/adr0001.rst [hunk 1]
```diff
@@ -16,8 +16,60 @@ ADR0001: Item library basis
...
+
+EasyReflectometryApp
+^^^^^^^^^^^^^^^^^^^^
+
+The application will commonly be referred to by users are EasyReflectometry, and indeed it should be usable without the understanding that there is a Python library foundation. 
+This said, the application project will be :mod:`EasyReflectometryApp` and should the application be installable via PyPI, the following command would be necessary. 
+
+.. code-block:: console
+
+    $ pip install EasyReflectometryApp
```
Comment (arm61) [inline docs/adrs/adr0001.rst:55]: This might be changed to 
```suggestion
    $ pip install EasyReflectometry
```

**Review by rozyczko — COMMENTED** [review]
Naming convention ADR is good. We should probably make this a general Easy Project ADR, though, since it should be universal for all Easy projects.

CI should probably not be an ADR, since it doesn't affect architecture per se and is/will be changing constantly (maybe?)

Item Library Basis is an implementation decision rather than architecture? Or maybe I don't understand what you want to cover by this ADR...

Bottom line: we need more ADRs for our projects

**Comment by rozyczko** [comment]
I like scipp's ADRs with their sectioning. Attributing the decision is important but for EasyReflectometry we know whom to blame, so maybe it's moot.
However, adding the `Consequences` section with negatives and positives would probably be beneficial since it will determine whether the ADR is worth adding.

**Comment by arm61** [comment]
I am using the phrase ADR to mean basically all decisions. i.e. for the CI we made a decision to release executables and python packages so I am recording that. So not limited to "architecture" (though one could argue that the CI ADR makes a decision about how or build process should be architected. 

I like a "consequences" section also. I will merge this and work on a consequences for each ADR produced.
